lab_2/main.py

```diff
@@ -13,7 +13,20 @@ def get_freq_dict(tokens: list) -> dict or None:
     :param tokens: a list of tokens
     :return: a dictionary with frequencies
     """
-    pass
+    if not isinstance(tokens, list):
+        return None
+
+    freq_dict = {}
+
+    for token in tokens:
+        if token in freq_dict:
+            freq_dict[token] += 1 / len(tokens)
```

Comment:
Is there really a need to calculate `len` of the `tokens` every time? May be save it in a separate variable?